ui/src/pages/authors/[id]/index.tsx

```diff
@@ -1,22 +1,29 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
```

Comment:
this feels like the e2e test should be updated, it's a very simple test at the moment and I know updating it won't be simple, is this functionality 'core' enough though to justify writing the test?

Reply:
We don't currently have any meaningful e2e tests for this page and there are some errors when running them.
<img width="844" alt="image" src="https://user-images.githubusercontent.com/48569671/204299842-1c227f51-1b03-4dea-a908-eff76a01c736.png">

Should I add e2e tests for this page in this PR?

Reply:
'no meaningful tests' suggests yes - if they won't take long , the existing ones were passing as few weeks ago...

Reply:
added/fixed e2e tests